hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerHAWithFailover.java

```diff
@@ -62,4 +70,48 @@ public void testIncrementalWaitTimeWithSameNodeFailover() throws Exception {
     Assert.assertEquals((numTimesTriedToSameNode + 1) * waitBetweenRetries,
         omFailoverProxyProvider.getWaitTime());
   }
+
```

Comment:
The javadoc for the test class, `TestOzoneManagerHAWithFailover` comments that tests should not be added to this class as it has a cleanup problem that leaves the cluster in a unusable states (not reusable).  Are we affected by this?

> * NOTE: Do not add new tests to this class since

https://github.com/apache/ozone/blob/40203afe63d40fa1340b64fd67ddfd5a2a6b0c99/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerHAWithFailover.java#L36

Reply:
The test has been moved to another place.